Mathlib/Algebra/Order/Monoid/Unbundled/Basic.lean

```diff
@@ -143,6 +143,12 @@ section Preorder
 
 variable [Preorder α]
 
+lemma mul_left_mono [CovariantClass α α (· * ·) (· ≤ ·)] {a : α} : Monotone (a * ·) :=
```

Comment:
Should these lemmas have additive counterparts? BTW, we already have [Monotone.const_mul'](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Monoid/Unbundled/Basic.html#Monotone.const_mul%27)

Reply:
Well spotted. Golfed